doc/releases/migration-guide-4.2.rst

```diff
@@ -500,6 +500,9 @@ OpenThread
 SPI
 ===
 
+* Renamed Renamed :kconfig:option:`CONFIG_SPI_MCUX_LPSPI` to :kconfig:option:`CONFIG_SPI_NXP_LPSPI`,
+  and similar for any child configs for that driver, including
+  :kconfig:option:`CONFIG_SPI_NXP_LPSPI_DMA` and :kconfig:option:`CONFIG_SPI_NXP_LPSPI_CPU`.
```

Comment:
Symbols that no longer exist can't use the `:kconfig:option:` directive.

```suggestion
* Renamed ``CONFIG_SPI_MCUX_LPSPI`` to :kconfig:option:`CONFIG_SPI_NXP_LPSPI`,
  and similar for any child configs for that driver, including
  :kconfig:option:`CONFIG_SPI_NXP_LPSPI_DMA` and :kconfig:option:`CONFIG_SPI_NXP_LPSPI_CPU`.
```

Reply:
I was wondering if it would show up like a red link or something and if that was desired